pygfx/renderers/wgpu/wgsl/mesh.wgsl

```diff
@@ -340,22 +340,22 @@ fn fs_main(varyings: Varyings, @builtin(front_facing) is_front: bool) -> Fragmen
     $$ endif
 
 
+    {$ include 'pygfx.uv_fragment.wgsl' $}
```

Comment:
I'm curious on the design choice here. Why use directly inlined code instead of letting the compiler handle this for you?

These should be trivial optimizations for it to unfold no?

Reply:
The main purpose is to make the code more modular.